package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.19.0",
```

Comment:
If you ever need to check and see if a dependency is installed, then you can look in this file.